tools/accuracy_checker/accuracy_checker/annotation_converters/style_transfer.py

```diff
@@ -35,7 +36,7 @@ def parameters(cls):
         return parameters
 
     def configure(self):
-        self.image_dir = self.get_value_from_config('images_dir')
+        self.image_dir = Path(self.get_value_from_config('images_dir'))
```

Comment:
Why is this needed? I thought `images_dir` being a `PathField` meant that the value would already be a `Path`.

Reply:
you are right, I deeper investigate issue and found that type casting never performend during getting value from config. native yml read it as string. when we use path without  prefixing in the config it saved as string